Types/TransactionType.cs

```diff
@@ -0,0 +1,24 @@
+using System.ComponentModel;
+
+namespace Types
+{
+    public enum TransactionType
+    {
+        [Description("Donatie")]
```

Comment:
I would stick with English until we have a request for providing internationalization